jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EJBTimerScheduler.java

```diff
@@ -125,16 +125,20 @@ public boolean removeJob(JobHandle jobHandle) {
 	
 	public TimerJobInstance getTimerByName(String jobName) {
 		for (Timer timer : timerService.getTimers()) {
-			Serializable info = timer.getInfo();
-			if (info instanceof EjbTimerJob) {
-				EjbTimerJob job = (EjbTimerJob) info;
-				
-				EjbGlobalJobHandle handle = (EjbGlobalJobHandle) job.getTimerJobInstance().getJobHandle();
-				if (handle.getUuid().equals(jobName)) {
-					logger.debug("Job  {} does match timer and is going to be returned", jobName);
-					return handle.getTimerJobInstance();
-				}
-			}
+		    try {
+    			Serializable info = timer.getInfo();
+    			if (info instanceof EjbTimerJob) {
+    				EjbTimerJob job = (EjbTimerJob) info;
+    				
+    				EjbGlobalJobHandle handle = (EjbGlobalJobHandle) job.getTimerJobInstance().getJobHandle();
+    				if (handle.getUuid().equals(jobName)) {
+    					logger.debug("Job  {} does match timer and is going to be returned", jobName);
+    					return handle.getTimerJobInstance();
+    				}
+    			}
+		    } catch (NoSuchObjectLocalException e) {
+                logger.debug("Timer info for {} was not found ", timer);
```

Comment:
Should this be logged with a level of WARN, instead of DEBUG? Or is this such a common occurrence that the user/administrator doesn't really need to know when it happens?